python3/lib/export.py

```diff
@@ -0,0 +1,61 @@
+import struct
+import operator
+from gltflib import (
+    GLTF, GLTFModel, Asset, Scene, Node, Mesh, Primitive, Attributes, Buffer, BufferView, Accessor, AccessorType,
+    BufferTarget, ComponentType, GLBResource, FileResource)
+
+from lib.parse_3db import Model
+from typing import List, Dict, Tuple
+
+def transform_point(p: Tuple[float, float, float]):
+    scale = 100
+    result = ((p[0] - 0.5) * scale, (p[1] -0.5) * scale, (p[2] - 0.5) * scale)
+    return result
+
+def build_vertices_array(triangles: List[int], points: List[Tuple[float, float, float]]):
+    vertices = [points[index] for index in triangles]
+    return vertices
+
+def export_to_gltf(model: Model):
+    mesh = model.meshes[0]
+    mesh_link = mesh.links[0]
```

Comment:
For completeness we need to handle multiple mesh links, although so far I've only ever seen meshes with a single mesh link.